packages/react-ui-components/src/SelectBox/selectBox.js

```diff
@@ -171,6 +171,10 @@ export default class SelectBox extends PureComponent {
     createNewEnabled = () => this.props.onCreateNew && this.props.searchTerm;
 
     optionsCount = () => {
+        if (this.props.options === undefined) {
```

Comment:
FYI I personally would not check against `undefined`, instead something like `Boolean(this.props.options) === false` is much safer